enterprise/audit/table.go

```diff
@@ -135,6 +135,7 @@ var auditableResourcesTypes = map[any]map[string]Action{
 		"created_by_name":         ActionIgnore,
 		"archived":                ActionTrack,
 		"source_example_id":       ActionIgnore, // Never changes.
+		"has_ai_task":             ActionIgnore,
```

Comment:
Probably worth tracking, no?

Reply:
Well, presumably at some point a template might change from not having an AI task to having one, and maybe back.

Reply:
A template version wouldn't though.